flutter_inappwebview_android/android/src/main/java/com/pichillilorenzo/flutter_inappwebview_android/webview/in_app_webview/InAppWebViewClient.java

```diff
@@ -299,12 +299,6 @@ public void onReceivedError(WebView view, @NonNull WebResourceRequest request, @
   public void onReceivedError(WebView view, int errorCode, String description, String failingUrl) {
     final InAppWebView webView = (InAppWebView) view;
 
-    if (!WebViewFeature.isFeatureSupported(WebViewFeature.SUPPRESS_ERROR_PAGE) &&
-            webView.customSettings.disableDefaultErrorPage) {
```

Comment:
You can just remove !WebViewFeature.isFeatureSupported(WebViewFeature.SUPPRESS_ERROR_PAGE)  and keep the if